### ISSUE TITLE
Type Casting using the <> operator

### ISSUE THREAD

**Issue by NabilShahid** [vue-styleguidist/vue-styleguidist#1450]
**Current behavior**

When i use typescript casting using <> operator, e,g. `const { checked } = <HTMLInputElement>event.target;`, i get `Unexpected token` error. I have installed Storybook which uses version `4.56.0` of `vue-docgen-api`.

**To reproduce**

Simply use this in any component logic
```
      const a='test';
      const b=<string>a
``` 
 
**Expected behavior**

It should not throw error when we cast to typescript type using <> operator.

**Comment by elevatebart** [comment]
It is recommended in TypeScript to avoid this notation that conflicts with TSX.

If you wish to keep the code as is, you will have to tell docgen that you did.

[Here is the config flag](https://github.com/vue-styleguidist/vue-styleguidist/blob/dev/packages/vue-docgen-api/src/parse.ts#L66). I clearly forgot to document it on the [api docs](https://vue-styleguidist.github.io/docs/Docgen.html#api)

If you find it within yourself to update the docs I would be forever grateful. TYTY

**Comment by elevatebart** [comment]
In storybook, it is not ver easy to configure though.
You will need to use the `webpackFinal` arg and update the `vue-docgen-loader` rule.
